README.md

```diff
@@ -808,6 +808,28 @@ By passing a special prop `adyenConfiguration` we are giving a possibility to cu
 }
 ```
 
+### <a id="security"></a><h2>Security</h2>
+
+Website security is a crucial thing nowadays and it is good to secure the website as much as it can be. Because part of the `mediastore-sdk` components library is a payment process, we wanna give you the possibility to make your website secure and safe.
+
+One of the possibilities to secure your website is [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) and it is also an [Adyen recommendation](https://docs.adyen.com/pt/development-resources/integration-security-guide#third-party-components) to do so.
+
+There are several ways to implement Content Security Policy. One of the way is to add the `<meta>` element in the header of the web application.
+Below you can find an example of the Content Security Policy which can be set the secure your website. This one exaplme allowing an application only to use resources that needed for woriking of the `mediastore-sdk`:
```

Comment:
exaplme -> example

Reply:
Last sentence fixes: This one example allows an application only to use resources that are needed for `mediastore-sdk` to work